addons/a11y/src/components/Report/index.js

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
@ndelangen if we want to use fragments, we need to specify `>=16.2.0` as react peer dependency